app/dao/organisation_dao.py

```diff
@@ -34,8 +34,9 @@ def dao_get_organisation_services(organisation_id):
 def dao_get_organisation_live_services(organisation_id):
     return Service.query.filter_by(
         organisation_id=organisation_id,
-        restricted=False
-    ).all()
+        restricted=False,
```

Comment:
Do we still need the check on restricted False? I don't think we can have something with a count_as_live being True AND having restricted False too so this line isn't really doing anything? Or are we just being extra cautious?